.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+<!-- A brief description of the PR here. -->
+
+## Adding a new package?
+
+If this pull request adds a new package:
+
+- [ ] The package's schema URL is correct.
+- [ ] The package metadata file, if present, contains: 
```

Comment:
I think this is good, but one thing to note is when it comes to the community contributions, I believe this is all automated, since all they do is open one PR to add the package to the package-list.json file. From that point forward the rest is an automated process. So the metadata file gets automatically generated and auto merged in these cases. So there might not be a way for us to catch these at PR time when it comes to community packages We could however validate these things using the schema file in the upstream repo since that is where the metadata file gets genned from when it comes to these cases.

Reply:
What I've been doing for these community-packages PRs -- and it's unfortunately pretty laborious -- is looking at the schema file and the repo and then checking that all of the necessary things (docs, logo, etc.) exist before approving the PR, for example here:

https://github.com/pulumi/registry/pull/3015

It's a painful process, but given the way we've got this set up now -- where the list-item PR comes through first, we merge that without much (any?) review, and then everything else after that that is automated and gets no review -- I'm not sure there's a much better option. At least not until we have some automation in place to check for the existence of some of these items before allowing a community-packages PR to be merged.